src/app/article-list.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { ArticleFormat } from './format';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ArticleListService {
+  public articleListData : ArticleFormat[];
+  constructor() { }
+
+  setArticleList(refrenceObject){
+    this.articleListData = refrenceObject;
+  }
+
+  getArticleList(){
```

Comment:
Add return type for functions. The same applies across all the other typescript files.

Reply:
Return type of _functions_ declared